CHANGELOG.md

```diff
@@ -4,14 +4,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-# v2.13.7
+# [Unreleased]
 
 ### Added
+- Addresses #175.
+- Introduced a new configuration variable `account_age` for setting a minimum account creation age.
+  - This utilizes `blocked` configuration variable to keep track of users under the limit.
```

Comment:
technical details aren't necessary

Reply:
Modified that message to make it less technical.